src/base/utils.js

```diff
@@ -85,6 +85,14 @@ export var Fullscreen = {
     } else if(document.msExitFullscreen) {
       document.msExitFullscreen()
     }
+  },
+  canGoFullscreen: function() {
+    return !(
+      document.fullscreenEnabled === false ||
+      document.webkitFullscreenEnabled === false ||
+      document.mozFullScreenEnabled === false ||
+      document.msFullscreenEnabled === false
+    )
```

Comment:
It is to make sure if they are all undefined it defaults to assuming full screen is available. Can change it though if we think one those properties will always exist?

> On 16 Jul 2016, at 00:37, Bruno Torres <notifications@github.com> wrote:
> 
> In src/base/utils.js:
> 
> > @@ -85,6 +85,14 @@ export var Fullscreen = {
> >      } else if(document.msExitFullscreen) {
> >        document.msExitFullscreen()
> >      }
> > +  },
> > +  canGoFullscreen: function() {
> > +    return !(
> > +      document.fullscreenEnabled === false ||
> > +      document.webkitFullscreenEnabled === false ||
> > +      document.mozFullScreenEnabled === false ||
> > +      document.msFullscreenEnabled === false
> > +    )
> Why don't you instead of this do:
> 
> return !!(document.fullscreenEnabled ||
>   document.webkitFullscreenEnabled ||
>   document.mozFullScreenEnabled ||
>   document.msFullscreenEnabled
> )
> I think it reads better, and avoids comparing to false.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>